### ISSUE TITLE
Port these views to VSCode?

### ISSUE THREAD

**Issue by mansona** [atom/github#2771]
## Summary

Ideally I would love to make use of the git and github tabs exactly as they worked in Atom but in VSCode

## Motivation

Now that [atom is being sunsetted 😭 ](https://github.blog/2022-06-08-sunsetting-atom/) I am trying to port all of my development processes across to VSCode and the one thing that I will miss the most is the git and github implementation in Atom. So I was wondering... how feasible would it be to port these views across to VSCode? I know the underlying structure of the editors are wildly different, but at least there is a common language of git here that could help?

If anyone has any insight into creating plugins for both editors I would love to know if this is even a reasonable suggestion 👍 

## Describe alternatives you've considered

I've tried to just use VSCode git integrations but it makes me cry when I think of what I'm losing 😭

**Comment by GEkuL** [comment]
Yeah, it would be great to have this exact addon on VS code and other platforms!

Default VS code git module sucks a lot ;(...

**Comment by xiang4444** [comment]
Maybe
